<launch>

  <arg name="project" default="lio_sam"/>

  <!-- ================= IMU TIME RECONSTRUCTION ================= -->
  <node pkg="lio_sam"
        type="imu_time_from_lidar_node"
        name="imu_time_from_lidar"
        output="screen">
    <param name="imu_in"    value="/vectornav/IMU"/>
    <param name="lidar_in"  value="/velodyne_points"/>
    <param name="imu_out"   value="/imu/data"/>
    <param name="imu_rate"  value="40.0"/>
  </node>

  <!-- ================= LIO-SAM PARAMETERS ================= -->
  <rosparam file="$(find lio_sam)/config/params_2velodyne_imu.yaml"
            command="load" />

  <!-- ================= lio_sam CORE ================= -->
  <include file="$(find lio_sam)/launch/include/module_loam.launch" />
  <include file="$(find lio_sam)/launch/include/module_robot_state_publisher.launch" />
  <include file="$(find lio_sam)/launch/include/module_rviz.launch" />

</launch>
